daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/TransactionVersion.scala

```diff
@@ -44,7 +44,8 @@ object TransactionVersion {
   def maxVersion: TransactionVersion = VDev
 
   // TODO(https://github.com/digital-asset/daml/issues/18240) move this feature flag to VDev.
-  private[lf] val minContractKeys = V31
+  private[lf] val minContractKeys = VDev
```

Comment:
Leftover from testing? (Done in https://github.com/digital-asset/daml/pull/18751)